templates/framework/doc_tree/view.html.twig

```diff
@@ -201,15 +201,15 @@
                                             Export
                                         </button>
                                         {# use first version if we need to edit the adoptionStatus #}
-                                        {% if editorRights %}
+                                        {% if isDraft %}
```

Comment:
One thing to note is that this will make it impossible to change the state after being changed to a non-draft state (ex. Adopted -> Deprecated).

Reply:
yeah i just noticed that, reworking...